### ISSUE TITLE
Home Assistant Supervisor does not seem to see the new release

### ISSUE THREAD

**Issue by BaQs** [marciogranzotto/addon-nightscout#10]
I'm running v1.1.1, I saw the 1.2 was released 2 days ago.
Unfortunately, somehow, the HA supervisor does not propose the upgrade.

Maybe some release file has to be updated ?

PS: thx for the integration! #wearenotwaiting

**Comment by marciogranzotto** [comment]
Yeah, the publish workflow is broken, I'll fix it today or tomorrow and re-release it

**Comment by BaQs** [comment]
thanks a lot!!

and thanks again for your fantastic job

**Comment by BaQs** [comment]
hello, it's me again :)

did you manage to fix the workflow ?

Best

**Comment by BaQs** [comment]
May I ask you if you got the time to fix the workflow ? 🙏
Of I can be of any help, please don't hesitate to ask 🙏

**Comment by BaQs** [comment]
Just got the notification to update the addon.
Big thanks !!!
